airbyte-webapp/src/views/Connection/ConnectionForm/ConnectionForm.tsx

```diff
@@ -155,19 +155,6 @@ const ConnectionForm: React.FC<ConnectionFormProps> = ({
   const initialValues = useInitialValues(connection, destDefinition, isEditMode);
   const workspace = useCurrentWorkspace();
 
-  const openResetDataModal = useCallback(() => {
-    openConfirmationModal({
-      title: "form.resetData",
-      text: "form.changedColumns",
-      submitButtonText: "form.reset",
-      cancelButtonText: "form.noNeed",
-      onSubmit: async () => {
-        await onReset?.();
-        closeConfirmationModal();
-      },
-    });
-  }, [closeConfirmationModal, onReset, openConfirmationModal]);
```

Comment:
:information_source: This was the old "should we reset or not" modal, that does not longer exist with this change.